nexus-config/src/nexus_config.rs

```diff
@@ -303,6 +304,11 @@ pub struct Tunables {
     ///
     /// If "None", nexus loops forever during initialization.
     pub load_timeout: Option<std::time::Duration>,
+
+    /// The amount of disk space to reserve for non-Crucible / control plane
+    /// storage in gibibytes. This amount represents a buffer that the region
```

Comment:
```suggestion
    /// The amount of space to reserve per-disk for non-Crucible storage (i.e., control plane
    /// storage) in gibibytes. This amount represents a buffer that the region
```

Trying to be really clear for people not already familiar with this.